### DISCUSSION TITLE
无法git clone 远程仓库

### DISCUSSION THREAD

**Discussion by zhaobuqi-01** [community/community#23285] [New to GitHub]
<p>git clone <a href="mailto:git@github.com">git@github.com</a>/zhaobuqi-01/photo.git hello<br>
fatal: repository ‘git@github.com/zhaobuqi-01/photo.git’ does not exist</p>

**Comment by lesleyanneb** [comment, marked as answer]
<p>Hi there,</p>
<p>Thank you for contributing to the GitHub community forum!</p>
<p>SSH URLs follow this format:</p>
<pre><code>git@github.com:USERNAME/REPOSITORY.git
</code></pre>
<p>If you want to clone the repository zhaobuqi-01/photo via SSH, you will need to use <a href="mailto:git@github.com">git@github.com</a>:zhaobuqi-01/photo with a colon rather than a slash.</p>
<p>I hope that helps to explain. Please let us know if you need anything else!</p>

**Comment by RajaDanyal** [comment]
<p><a href="https://medium.com/@danyalraja/easy-start-git-using-command-line-d95f8eb559d4" rel="nofollow noopener">https://medium.com/@danyalraja/easy-start-git-using-command-line-d95f8eb559d4</a></p>
